### PR TITLE
[TACHYON-1462] Document security related properties in Configuration-Settings.md

### PR DESCRIPTION
**Pull request by dongche**
JIRA: https://tachyon.atlassian.net/browse/TACHYON-1462
          https://tachyon.atlassian.net/browse/TACHYON-1677

Doc security related properties in Configuration-Settings.md.
Doc security features in Security.md.

@apc999 , @yupeng9 , @jsimsa , @sundapeng , would you like to take a look? Your feedbacks are appreciated to improve the doc quality. Thanks!

BTW, I will install Jekyll to generate the doc html and verify the links could navigate well tomorrow.

### CODE REVIEW

**Comment by apc999** [comment]
thanks 
LGTM

File: docs/en/Configuration-Settings.md [hunk 1]
```diff
@@ -746,6 +747,68 @@ configuration options.
...
+  the created directory has permission 755 and file has permission 644.</td>
+</tr>
+<tr>
+  <td>tachyon.security.authorization.permission.supergroup</td>
+  <td>supergroup</td>
+  <td>The super group of Tachyon file system. All users in this group have super permission.</td>
+</tr>
+<tr>
+  <td>tachyon.security.group.mapping.class</td>
+  <td>tachyon.security.group.provider.ShellBasedUnixGroupsMapping</td>
```
Comment (sundapeng) [inline docs/en/Configuration-Settings.md]: It's better to break here by adding **{& # 8 2 0 3 ; }** after **tachyon.security.group.provider.**

File: docs/en/Configuration-Settings.md [hunk 2]
```diff
@@ -29,11 +29,12 @@ distributed with Tachyon binaries. We do not recommend beginner users to edit th
...
-All Tachyon configuration properties fall into one of the five categories:
+All Tachyon configuration properties fall into one of the six categories:
 [Common](#common-configuration) (shared by Master and Worker),
-[Master specific](#master-configuration), [Worker specific](#worker-configuration), and
-[User specific](#user-configuration), and [Cluster specific](#cluster-management) (used for running
-Tachyon with cluster managers like Mesos and YARN).
+[Master specific](#master-configuration), [Worker specific](#worker-configuration),
+[User specific](#user-configuration), [Cluster specific](#cluster-management) (used for running
+Tachyon with cluster managers like Mesos and YARN), and [Security specific]
+(#security-configuration) (shared by Master, Worker, and User).
```
Comment (sundapeng) [inline docs/en/Configuration-Settings.md]: Please move **(#security-configuration)** the end of previous line, otherwise it wouldn't be a link.

**Comment by sundapeng** [comment]
Hi Dong, I verified it using Jekyll. Two minor comments left, others are LGTM.

**Comment by dongche** [comment]
@sundapeng , @apc999 , thanks for your review and verification! Patch updated.

**Comment by dongche** [comment]
Jenkins, test this please

**Comment by sundapeng** [comment]
LGTM, thank Dong for updating the patch.

**Comment by apc999** [comment]
LGTM. thanks for the patch @dongche  and thanks for the review @sundapeng

**Pull request merged**